### ISSUE TITLE
Hebrew text should be RTL but displayed LTR

### ISSUE THREAD

**Issue by avivhu** [github/copilot-cli#3704]
### Describe the bug

Copilot CLI doesn't display RTL languages correctly. The problem is present for Hebrew and Arabic.
PrompT: how do you say hello in hebrew? write it in text

Response word appears flipped LTR. 

<img width="258" height="72" alt="Image" src="https://github.com/user-attachments/assets/fd2a6c07-8ae3-4286-a954-cb9b494dfc0f" />

In contrast Claude Code text mode writes it correctly.

<img width="1127" height="181" alt="Image" src="https://github.com/user-attachments/assets/93af6016-74e5-4c85-bd1b-db1eee61b997" />

### Affected version

GitHub Copilot CLI 1.0.60.

### Steps to reproduce the behavior

1. Set /model to Claude Opus 4.8
2. Type "how do you say hello in hebrew? write it in text"
3.  The output displays the Hebrew characters in text in flipped order

### Expected behavior

The word Shalom should appear like this

<img width="299" height="28" alt="Image" src="https://github.com/user-attachments/assets/29abb6ff-baec-4ac6-9a19-4aefa879e71f" />

**Comment by yarden1423** [comment]
any progress?
